flink-end-to-end-tests/test-scripts/test_kubernetes_pyflink_application.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+source "$(dirname "$0")"/common_kubernetes.sh
+
+CURRENT_DIR=`cd "$(dirname "$0")" && pwd -P`
+CLUSTER_ROLE_BINDING="flink-role-binding-default"
+CLUSTER_ID="flink-native-k8s-pyflink-application-1"
+PURE_FLINK_IMAGE_NAME="test_kubernetes_application"
+PYFLINK_IMAGE_NAME="test_kubernetes_pyflink_application"
+LOCAL_LOGS_PATH="${TEST_DATA_DIR}/log"
+
+function internal_cleanup {
+    kubectl delete deployment ${CLUSTER_ID}
+    kubectl delete clusterrolebinding ${CLUSTER_ROLE_BINDING}
+}
+
+start_kubernetes
+
+build_image ${PURE_FLINK_IMAGE_NAME}
+
+# Build PyFlink wheel package
```

Comment:
We have to make sure that this serious of commands to build an image for python is stable. Otherwise, the devs will suffer a lot from this failure.

Do you know how long it will take to run this E2E test?

Reply:
Yes, the e2e test will first build a flink base image using the local flink binary, and then it will turn to the local `flink-python` module to build a pyflink dist which will be added and installed into the new dedicated pyflink image. And then run the e2e test with the new image. The overall elapsed time is about 10 minutes, most of time is for building the image.